addons/website_sale/controllers/main.py

```diff
@@ -190,18 +177,15 @@ def _shop_lookup_products(self, attrib_set, options, post, search, website):
 
         return fuzzy_search_term, product_count, search_result
 
-    def _shop_get_query_url_kwargs(
-        self, category, search, min_price, max_price, attrib=None, order=None, tags=None, **post
-    ):
-        return {
-            'category': category,
-            'search': search,
-            'attrib': attrib,
-            'tags': tags,
-            'min_price': min_price,
-            'max_price': max_price,
-            'order': order,
-        }
+    def _shop_get_query_url_kwargs(self, **kwargs):
+        vals = {}
+        ignore_args = [
+            'attribute_value',
+        ]
+        for kw, value in kwargs.items():
+            if value and kw not in ignore_args:
+                vals[kw] = value
+        return vals
```

Comment:
Mhm, I prefer to keep a dict of specific values instead of saying 'we take everything but that one'
Can't you simply stop considering `attrib` and leave the rest as it was?